### PR TITLE
backport: v0.20 pr1

### PR DESCRIPTION
**Pull request by Munkybooty**

### CODE REVIEW

File: doc/bips.md [hunk 1]
```diff
@@ -18,17 +18,18 @@ BIPs that are implemented by Dash Core (up-to-date up to **v18.0**):
...
 * [`BIP 65`](https://github.com/bitcoin/bips/blob/master/bip-0065.mediawiki): The CHECKLOCKTIMEVERIFY softfork was merged in **v0.12.0** ([PR #6351](https://github.com/bitcoin/bitcoin/pull/6351)), and backported to **v0.11.2** and **v0.10.4**. Mempool-only CLTV was added in [PR #6124](https://github.com/bitcoin/bitcoin/pull/6124).
 * [`BIP 66`](https://github.com/bitcoin/bips/blob/master/bip-0066.mediawiki): The strict DER rules and associated version 3 blocks have been implemented since **v0.10.0** ([PR #5713](https://github.com/bitcoin/bitcoin/pull/5713)).
-* [`BIP 68`](https://github.com/bitcoin/bips/blob/master/bip-0068.mediawiki): Sequence locks have been implemented as of **v0.12.1**  ([PR #7184](https://github.com/bitcoin/bitcoin/pull/7184)), and have been activated since *block 419328*.
+* [`BIP 68`](https://github.com/bitcoin/bips/blob/master/bip-0068.mediawiki): Sequence locks have been implemented as of **v0.12.1**  ([PR #7184](https://github.com/bitcoin/bitcoin/pull/7184)), and have been *buried* since **v0.19.0** ([PR #16060](https://github.com/bitcoin/bitcoin/pull/16060)).
```
Comment (UdjinM6) [inline doc/bips.md:21]: 17111: requires 16060, drop

File: test/functional/feature_block.py [hunk 2]
```diff
@@ -45,7 +45,7 @@
     OP_RETURN,
     OP_TRUE,
     SIGHASH_ALL,
-    SignatureHash,
+    LegacySignatureHash,
```
Comment (UdjinM6) [inline test/functional/feature_block.py:48]: 17141: ok-ish but maybe DNM would be better

**Review by UdjinM6 — CHANGES_REQUESTED** [review]
pls see below + https://github.com/UdjinM6/dash/commits/pr4821

**Review by PastaPastaPasta — APPROVED** [review]
utACK for merging via merge commit

**Review by UdjinM6 — APPROVED** [review]
utACK

(nit: could squash fixes)

**Review by PastaPastaPasta — APPROVED** [review]
re-utACK

**Review by UdjinM6 — APPROVED** [review]
utACK

**Pull request merged**